scripts/variantstore/wdl/GvsCreateVATAnnotations.wdl

```diff
@@ -0,0 +1,320 @@
+        call AnnotateVCF {
+          input:
+            input_vcf = ExtractAnAcAfFromVCF.output_vcf,
+            input_vcf_index = ExtractAnAcAfFromVCF.output_vcf_index,
+            output_annotated_file_name = "${input_vcf_name}_annotated",
+            nirvana_data_tar = nirvana_data_directory,
+            custom_annotations_file = ExtractAnAcAfFromVCF.annotations_file,
+        }
+
+        call PrepAnnotationJson {
+          input:
+            annotation_json = AnnotateVCF.annotation_json,
+            output_file_suffix = "${input_vcf_name}.json.gz",
+            output_path = output_path,
+            service_account_json_path = service_account_json_path
+        }
+
+        # ------------------------------------------------
+        # Outputs:
+        output {
+            Int count_variants = ExtractAnAcAfFromVCF.count_variants
+            File track_dropped = ExtractAnAcAfFromVCF.track_dropped
+            Boolean done = true
+        }
+}
+
+################################################################################
+
+task ExtractAnAcAfFromVCF {
+    input {
+        File input_vcf
+        File input_vcf_index
+        String? service_account_json_path
+        File subpopulation_sample_list
+        File custom_annotations_template
+        File ref
+        String output_path
+    }
+    parameter_meta {
+        input_vcf: {
+          localization_optional: true
+        }
+        input_vcf_index: {
+          localization_optional: true
+        }
+    }
+
+    String has_service_account_file = if (defined(service_account_json_path)) then 'true' else 'false'
+    String custom_annotations_file_name = "ac_an_af.tsv"
+    String local_input_vcf = basename(input_vcf)
+    String local_input_vcf_index = basename(input_vcf_index)
+    String normalized_vcf = "normalized.vcf"
+    String normalized_vcf_compressed = "normalized.vcf.gz"
+    String normalized_vcf_indexed = "normalized.vcf.gz.tbi"
+
+    # separate multi-allelic sites into their own lines, remove deletions and filtered sites and make a sites only vcf
+    # while extracting and calculating the an/ac/af & sc by subpopulation into a tsv
+    command <<<
+        set -e
+
+        if [ ~{has_service_account_file} = 'true' ]; then
+          gsutil cp ~{service_account_json_path} local.service_account.json
+          export GOOGLE_APPLICATION_CREDENTIALS=local.service_account.json
+          gcloud auth activate-service-account --key-file=local.service_account.json
+
+        fi
+
+        cp ~{custom_annotations_template} ~{custom_annotations_file_name}
+
+        gsutil cp ~{input_vcf} ~{local_input_vcf}
+        gsutil cp ~{input_vcf_index} ~{local_input_vcf_index}
+        gsutil cp ~{ref} Homo_sapiens_assembly38.fasta
+
+        ## compare the ancestry sample list with the vcf sample list
+        # TODO throw an error (but dont fail the job) if there are samples that are in one, but not the other. Throw two different errors.
```

Comment:
Can you create a ticket for this and take out the TODO?

Reply:
https://broadworkbench.atlassian.net/browse/VS-292